backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/DayUtil.kt

```diff
@@ -1,27 +1,26 @@
 package app.ehrenamtskarte.backend.verification
 
-import java.time.LocalDateTime
-import java.time.ZoneOffset
-import java.time.temporal.ChronoUnit
-import java.time.temporal.TemporalUnit
+import java.time.Clock
+import java.time.LocalDate
+import java.time.Month
 
 class DayUtil {
     companion object {
-        fun isOnOrBeforeToday(maxInclusiveDay: LocalDateTime): Boolean {
-            // FIXME: verify this and write unit test
-            return !maxInclusiveDay.isAfter(LocalDateTime.now().truncatedTo(ChronoUnit.DAYS))
+        fun isOnOrBeforeToday(maxInclusiveDay: LocalDate, clock: Clock?): Boolean {
```

Comment:
We should probably pass a timezone here to evaluate what `today` means and to compare that with `maxInclusiveDay`